exportcontrol/settings.py

```diff
@@ -21,9 +11,11 @@
 
 # https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
 SECRET_KEY = os.getenv("SECRET_KEY")
-DEBUG = os.getenv("DEBUG", default="false") == "true"
 
-ALLOWED_HOSTS = ["0.0.0.0:8000"]
+DEBUG = os.getenv("DEBUG", "False") == "True"
```

Comment:
This is not the Python `True`/`False`, it's the value of these environment variables in `.env` (or system env vars). The convention here is lower-case `true`/`false`, so I think we should keep it as it is.
```suggestion
DEBUG = os.getenv("DEBUG", "false") == "true"
```

Reply:
I agree, but that doesn't work